packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -84,7 +200,7 @@ class ObjectsInRedisClient {
 
         if (!protoVersion) {
             // if no proto version existent yet, we set ours
-            const highestVersion = Math.max(...this.supportedProtocolVersions);
+            const highestVersion = Math.max(...this.supportedProtocolVersions.map(value => parseInt(value)));
```

Comment:
Currently, the helper methods get the protocol version from the db as string (also in states client). Now the checks are performing just `supportedProtocolVersions.includes`, it would be easy to change to `includes(parseInt)`. However, it does not necessarily have to be part of this PR from my point of view.

Reply:
Yeah I was just confused that there are like 5 usages of supportedProtocolVersions, 4 of which are either defining it as a string or converting it to/from a string, while the 1 that actually works with it would benefit from it being a number.